### ISSUE TITLE
Inconsistent PIV driver name

### ISSUE THREAD

**Issue by mouse07410** [OpenSC/OpenSC#818]
### Expected behaviour

Based on the output of `opensc-tool --list-drivers`, I should be able to use the name `piv` when configuring OpenSC via `opensc.conf` file, editing/adding commands like `card_drivers = openpgp, piv, internal;`, etc.

```
$ opensc-tool --list-drivers
Configured card drivers:
  piv              PIV-II  for multiple cards
  openpgp          OpenPGP card
  cardos           Siemens CardOS
  flex             Schlumberger Multiflex/Cryptoflex
  cyberflex        Schlumberger Cyberflex
  gpk              Gemplus GPK
  gemsafeV1        driver for the Gemplus GemSAFE V1 applet
  miocos           MioCOS 1.1
  asepcos          Athena ASEPCOS
  starcos          STARCOS SPK 2.3/2.4/3.4
  tcos             TCOS 3.0
  jcop             JCOP cards with BlueZ PKCS#15 applet
  oberthur         Oberthur AuthentIC.v2/CosmopolIC.v4
  authentic        Oberthur AuthentIC v3.1
  iasecc           IAS-ECC
  belpic           Belpic cards
  ias              IAS
  incrypto34       Incard Incripto34
  acos5            ACS ACOS5 card
  akis             TUBITAK UEKAE AKIS
  entersafe        entersafe
  epass2003        epass2003
  rutoken          Rutoken driver
  rutoken_ecp      Rutoken ECP driver
  westcos          WESTCOS compatible cards
  myeid            MyEID cards with PKCS#15 applet
  sc-hsm           SmartCard-HSM
  dnie             DNIe: Spanish eID card
  MaskTech         MaskTech Smart Card
  mcrd             MICARDO 2.1 / EstEID 1.0 - 3.0
  setcos           Setec cards
  muscle           MuscleApplet
  atrust-acos      A-Trust ACOS cards
  itacns           Italian CNS
  isoApplet        Javacard with IsoApplet
  gids             GIDS Smart Card
  default          Default driver for unknown cards
```

### Actual behaviour

Instead, it does not seem to work unless I use the name `PIV-II` instead of `piv`, so the above command in `opensc.conf` has to look like `card_drivers = openpgp, PIV-II, internal;`

### Steps to reproduce

1. Build and install the current master of OpenSC
2. Edit `opensc.conf` restricting the allowed drivers to PIV and OpenPGP using name `piv` as `opensc-tool --list-drivers` says
3. Observe that you cannot use tools with PIV cards because the PIV driver is not loaded.

**Comment by frankmorgner** [comment]
Yes, this should work as you say. If you want to have a look yourself, you need to synchronize `internal_card_drivers` in `src/libopensc/ctx.c` with every driver's `short_name` (in your case this would be in `src/libopensc/card-piv.c`)...

**Comment by mouse07410** [comment]
Please see https://github.com/OpenSC/OpenSC/pull/825

**Comment by viktorTarasov** [comment]
resolved with #828 -- short name changed to _PIV-II_

**Comment by mouse07410** [comment]
Thank you - this is a better way. I was going to change to it myself - you've beaten me to it. ;)
